### ISSUE TITLE
Add ability to organize the Dashboard listing

### ISSUE THREAD

**Issue by agilliland** [metabase/metabase#1150]
Talking with Rudy recently and their setup has about 15-20 dashboards in it and it's getting hard to parse through that list all the time.  The hope would be to have some way of moving the higher profile dashboards up on the list or some other way to pin important dashboards.

**Comment by salsakran** [comment]
@kdoh let's link this to the Dashboard Organization issue once we get there.

**Comment by camsaul** [comment]
@metabase/core-developers can we close this out? I made some changes to return the Dashboards list in case-insensitive alphabetical order a while back and we haven't heard anything about this since. We can always open a new issue if a have an actionable way to improve dashboards organization.

**Comment by salsakran** [comment]
still a very real problem, I'm afraid.

**Comment by tanzoniteblack** [comment]
We have several different departments utilizing information from metabase. The collections feature has been fantastic for organizing questions, but dashboards are beginning to get unwieldy. Perhaps being able to add dashboards to collections would be enough of an organization method?

**Comment by tlrobinson** [comment]
Superseded by #2326
